### ISSUE TITLE
Number of Postman API Request Warn

### ISSUE THREAD

**Issue by suhagunduz** [a85/POSTMan-Chrome-Extension#11529]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

Using the Authorization key, we establish a Postman collection and connect using the Postman CLI. Through Postman, we wish to keep an eye on the APIs we manage. We do not receive a warning when the maximum number of Postman API requests is reached; instead, the CLI tries to run again and encounters an issue.

![image](https://user-images.githubusercontent.com/59437172/206470277-30102e31-784a-472d-9bfd-066a206968e8.png)

### Describe the solution you'd like

A warning e-mail will be sent to users when the maximum usage limit is reached as a result of CLI usage, making users aware of the situation.

**Comment by ozcanoguz** [comment]
Will watch that as we have faced with such a problem as well, we were experimenting Postman tool and reached API key limits but there was no "monitoring" mechanism informed us that we are about to reach our limits.

As postman already contains "e-mails" of teams that is working on workspaces, I would say we should be alerted for that notice even it is freely or commercially used.

On the other hand, same does not apply once we click "Purchase" button and do not buy licence of the software, today I have retrieved "Hey <username> was there an issue with your Postman purchase?" titled e-mail which monitors for marketing but not functions of the tool.

Would be good to add this to product or have an explanation how this procedure is applied.
